scripts/build_minikube.sh

```diff
@@ -107,6 +107,9 @@ else
   NODE_ENV=$NODE_ENV
 fi
 
+
+./generate-certs.sh
```

Comment:
You also need to do this for `scripts/build_microk8s.sh`

Reply:
Added for microk8s